caluma/tests/__snapshots__/test_schema.ambr

```diff
@@ -82,6 +82,58 @@
     clientMutationId: String
   }
   
+  type AggregateField implements Node {
+    createdAt: DateTime!
+    modifiedAt: DateTime!
+    createdByUser: String
+    createdByGroup: String
+    modifiedByUser: String
+    modifiedByGroup: String
+  
+    """The ID of the object"""
+    id: ID!
+    sourceField: AnalyticsField!
+    function: CalumaAnalyticsAggregateFieldFunctionChoices!
+    alias: String!
+    table: PivotTable!
+    meta: GenericScalar
+    filters: [String]
+  }
+  
+  type AggregateFieldConnection {
+    """Pagination data for this connection."""
+    pageInfo: PageInfo!
+  
+    """Contains the nodes in this connection."""
+    edges: [AggregateFieldEdge]!
+    totalCount: Int
+  }
+  
+  """A Relay edge containing a `AggregateField` and its cursor."""
+  type AggregateFieldEdge {
+    """The item at the end of the edge"""
+    node: AggregateField
+  
+    """A cursor for use in pagination"""
+    cursor: String!
+  }
+  
+  input AggregateFieldFilterSetType {
```

Comment:
This name looks like the artefact of a django generic `related_name`. consider explicitly setting `related_name` kwarg in the field definition assignment.

Reply:
remodelled